packages/devextreme/js/__internal/core/state_manager/interfaces.ts

```diff
@@ -0,0 +1,163 @@
+export type IStateManagerActionType = 'UPDATE' | 'INITIALIZE' | 'CLEAR_HISTORY';
+
+export interface IStateChangePayload {
```

Comment:
Please, do not use leading `I` for interfaces. AFAIKS, Hungarian notation isn’t used for interfaces anywhere in the codebase. (I've searched for `(interace|type) I[A-Z]` in the devextreme package, skipping generated code)

```suggestion
export interface StateChangePayload {
```

Reply:
`I` for other types other then intrefaces is also unnecessary, I mean the `IStateManagerActionType`above

Reply:
The file should be renamed to `types.ts` (There is a number of other files in the `__internal` folder)

Reply:
JFYI: to deal with name conflicts you can use aggregated exports:

`export type * as StateManagementTypes from './types'`
or
`import type * as StateManagementTypes from './types'`

Reply:
Changed